src/utils/log.ts

```diff
@@ -1,57 +1,64 @@
-import { stderr as chalkStderr } from "chalk";
-import { CliKintoneError } from "./error";
+import winston from "winston";
 
-const currentISOString = () => new Date().toISOString();
+const { combine, timestamp, printf, splat } = winston.format;
 
-export type Logger = {
-  debug: (message: any) => void;
-  info: (message: any) => void;
-  warn: (message: any) => void;
-  error: (message: any) => void;
-  fatal: (message: any) => void;
-};
+export type LogLevel = "debug" | "info" | "warn" | "error" | "fatal";
+export type Logger = winston.Logger &
+  Record<keyof typeof logLevels.levels, winston.LeveledLogMethod>;
+export const SUPPORTED_LOG_LEVELS: LogLevel[] = [
+  "debug",
+  "info",
+  "warn",
+  "error",
+  "fatal",
+];
 
-export const logger: Logger = {
-  debug: (message: any) => {
-    const prefix = `[${currentISOString()}] ${chalkStderr.green("DEBUG")}:`;
-    console.error(addPrefixEachLine(message, prefix));
-  },
+export const logFormat = printf((info) => {
+  info.level = info.level.toUpperCase();
+  const transformedInfo = winston.format.colorize().transform(info, {
+    level: true,
+  }) as winston.Logform.TransformableInfo;
 
-  info: (message: any) => {
-    const prefix = `[${currentISOString()}] ${chalkStderr.blue("INFO")}:`;
-    console.error(addPrefixEachLine(message, prefix));
-  },
+  if (!transformedInfo) {
+    return "";
+  }
 
-  warn: (message: any) => {
-    const prefix = `[${currentISOString()}] ${chalkStderr.yellow("WARN")}:`;
-    console.error(addPrefixEachLine(message, prefix));
-  },
+  return `[${transformedInfo.timestamp}] ${transformedInfo.level}: ${transformedInfo.message}`;
+});
 
-  error: (message: any) => {
-    const parsedMessage = parseErrorMessage(message);
-    const prefix = `[${currentISOString()}] ${chalkStderr.red("ERROR")}:`;
-    console.error(addPrefixEachLine(parsedMessage, prefix));
+const logLevels = {
+  levels: {
+    fatal: 0,
+    error: 1,
+    warn: 2,
+    info: 3,
+    debug: 4,
   },
-
-  fatal: (message: any) => {
-    const prefix = `[${currentISOString()}] ${chalkStderr.bgRed("FATAL")}:`;
-    console.error(addPrefixEachLine(message, prefix));
+  colors: {
+    fatal: "redBG",
+    error: "red",
+    warn: "yellow",
+    info: "blue",
+    debug: "green",
   },
 };
 
-const addPrefixEachLine = (message: any, prefix: string): string =>
-  ("" + message)
-    .split("\n")
-    .filter((line) => line.length > 0)
-    .map((line) => `${prefix} ${line}`)
-    .join("\n");
-
-const parseErrorMessage = (error: unknown): string => {
-  if (error instanceof Error) {
-    if (error instanceof CliKintoneError) {
-      return error.toString();
-    }
-    return "" + error;
```

Comment:
MUST: If the log message has linebreaks, each line should have prefixes.

```
# With error message: "The first line\nThe second line\nThe third line"

## Current and expected
[2023-09-14T19:58:55.938Z] ERROR: The first line
[2023-09-14T19:58:55.938Z] ERROR: The second line
[2023-09-14T19:58:55.938Z] ERROR: The third line

## New
[2023-09-14T19:58:55.938Z] ERROR: The first line
The second line
The third line
```

It is missing from our spec, but please keep this behavior.
I will add it to the spec later.